gitlabform/processors/project/integrations_processor.py

```diff
@@ -9,14 +10,19 @@ def __init__(self, gitlab: GitLab):
         super().__init__("integrations", gitlab)
 
     def _process_configuration(self, project_and_group: str, configuration: dict):
-        for integration in sorted(configuration["integrations"]):
-            if configuration.get("integrations|" + integration + "|delete"):
+        configured_integrations = configuration.get("integrations", {})
+        project: Project = self.gl.projects.get(project_and_group)
```

Comment:
use `python_gitlab` cached method here?

Reply:
Thanks @TimKnight-DWP for the review and reminder. Have been meaning to update and use the cached method but that change wasn't merged at the time when I last looked at it. I just updated the PR. Could you please do another review?